src/orc.jl

```diff
@@ -0,0 +1,48 @@
+export OrcJIT, OrcModule, dispose, compile, address, addressin
+
+@checked struct OrcJIT 
+    ref::API.LLVMOrcJITStackRef
+end
+
+Base.unsafe_convert(::Type{API.LLVMOrcJITStackRef}, orc::OrcJIT) = orc.ref
+
+"""
+    ORC(::TargetMachine)
+
+Takes ownership of the provided target machine.
+"""
+function OrcJIT(tm::TargetMachine)
+    OrcJIT(API.LLVMOrcCreateInstance(tm))
+end
+
+function dispose(orc::OrcJIT)
+    API.LLVMOrcDisposeInstance(orc)
+end
+
+struct OrcModule 
+    ref::API.LLVMOrcModuleHandle
+end
+Base.cconvert(::Type{API.LLVMOrcModuleHandle}, mod::OrcModule) = mod
```

Comment:
Why is this needed?

Reply:
OrcModuleHandle is just an UInt64. And not an ValueRef.